discojs/discojs-core/package.json

```
@@ -22,6 +22,7 @@
   "dependencies": {
     "@tensorflow/tfjs": "4",
     "@types/msgpack-lite": "0.1",
+    "@xenova/transformers": "^2.16.1",
```

`npm i --save` likes to add the full version, but I do prefer to use the package-lock.json to keep track of the actual version we're supporting and keep package.json will only the major version

```suggestion
    "@xenova/transformers": "2",
```

I just noticed that "gpt-tokenizer" is also a dep, but was never used. can you also remove it?